src/main/java/org/broadinstitute/hellbender/exceptions/UserException.java

```diff
@@ -427,4 +427,12 @@ public CouldNotIndexFile(final File file, final Exception e) {
                     file.getAbsolutePath(), e.getClass().getCanonicalName(), e.getMessage()), e);
         }
     }
+
+    public static final class UnimplementedFeature extends UserException {
+        private static final long serialVersionUID = 0L;
+
+        public UnimplementedFeature(String message){
+            super(message);
```

Comment:
New `UserException` subclasses should generally prepend some boilerplate text to the error message in their constructors (eg, "Feature is not currently implemented: %s")

Reply:
I feel like this generally leads to more confusing error messages since you have to fight the existing text to get it to say what you want.

Reply:
It leads to more consistent error messages. You don't have to "fight" the existing text so much as just look at it before adding your message.